src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,12 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed
+    ? 'TodoInfo--completed'
+    : ''}`}
```

Comment:
let's use [classnames library](https://github.com/JedWatson/classnames) for conditional rendering classes